src/explorer.api/Explorers/RealColumnExplorer.cs

```diff
@@ -12,52 +12,157 @@
 
     internal class RealColumnExplorer : ColumnExplorer
     {
+        // TODO: The following should be configuration items (?)
+        private const long ValuesPerBucketTarget = 20;
+
+        private const double SuppressedRatioThreshold = 0.1;
+
         public RealColumnExplorer(JsonApiClient apiClient, ExploreParams exploreParams)
             : base(apiClient, exploreParams)
         {
+            ExploreMetrics = Array.Empty<ExploreResult.Metric>();
         }
 
-        public override async IAsyncEnumerable<ExploreResult> Explore()
+        public IEnumerable<ExploreResult.Metric> ExploreMetrics { get; set; }
+
+        public override async Task Explore()
```

Comment:
Yes and yes. It's basically a cut and paste... (And I repeated it again for Bool and Text explorers).

Initially I thought the implementations might diverge more. Some or all of these items can probably be moved to the `ColumnExplorer` abstract base class. The `Explore` task can also be refactored a lot to avoid repetition, maybe some parts can be factored out into `DiffixUtilities` (btw. that's a crappy name, I know). 

In any case, I think the `ColumnExplorer` could do with a few changes to the way the metrics are collected and reported. (#48)

Reply:
Should we add another issue for refactoring the common code now?

Reply:
I created #53 to track this.